src/OmniSharp.Roslyn.CSharp/Services/Intellisense/CompletionItemExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Completion;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Intellisense
+{
+    internal static class CompletionItemExtensions
+    {
+        private static MethodInfo _getSymbolsAsync;
+
+        static CompletionItemExtensions()
+        {
+            var symbolCompletionItemType = typeof(CompletionItem).GetTypeInfo().Assembly.GetType("Microsoft.CodeAnalysis.Completion.Providers.SymbolCompletionItem");
+            _getSymbolsAsync = symbolCompletionItemType.GetMethod("GetSymbolsAsync", BindingFlags.Public | BindingFlags.Static);
```

Comment:
We've got some extension methods for this sort of thing (used by [`MetadataHelper`](https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Roslyn/MetadataHelper.cs)) that throw exceptions nicely when they fail. I'd recommend using those to ensure we can easily spot failures if internal API names change.